proto/viam/app/datasync/v1/data_sync.proto

```diff
@@ -76,55 +77,12 @@ message CaptureInterval {
 message DataCaptureMetadata {
   string component_type = 1;
   string component_name = 2;
-  string component_model = 3;
   string method_name = 4;
   DataType type = 5;
   map<string, google.protobuf.Any> method_parameters = 6;
   string file_extension = 7;
   repeated string tags = 8;
   string session_id = 9;
-}
-
-message TabularCapture {
```

Comment:
Replaced in app months ago with [dbBinaryData](https://github.com/viamrobotics/app/blob/4e313dba532bb2bb2a817027f1c94bc08ecaa362/datasync/binary_db.go#L39-L63) and [dbTabularData](https://github.com/viamrobotics/app/blob/4e313dba532bb2bb2a817027f1c94bc08ecaa362/datasync/tabular_db.go#L36-L59)